.github/workflows/slovensko_digital_ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Slovensko.Digital CI
+
+on: 
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    if: ${{github.ref == 'refs/heads/master'}}
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        upvs_env: [dev, fix, prod]
```

Comment:
Matrix v tomto pripade nema zmysel, lebo tu to myslim ide len na production.

Reply:
ano, pri tomto projekte to neriesime